hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/DirScanPool.java

```diff
@@ -109,11 +117,21 @@ synchronized void tryUpdatePoolSize(long timeout) {
         break;
       }
     }
-    LOG.info("Update chore's pool size from {} to {}", pool.getPoolSize(), size);
+    LOG.info("Update {} chore's pool size from {} to {}", name, pool.getPoolSize(), size);
     pool.setCorePoolSize(size);
   }
 
   public int getSize() {
     return size;
   }
+
+  public static DirScanPool getArchiveScanner(Configuration conf) {
```

Comment:
GetScanPool instead of getScanner? Scanner has special meanings in HBase which may confuse developers.

Reply:
+1. Got the same Q when read the method name at begin of patch.